src/explore-education-statistics-admin/src/pages/publication/components/__tests__/PublicationAddExistingUsersTab.test.tsx

```diff
@@ -0,0 +1,128 @@
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import React from 'react';
+import PublicationAddExistingUsersTab from '@admin/pages/publication/components/PublicationAddExistingUsersTab';
+import _releasePermissionService, {
+  ManageAccessPageContributor,
+} from '@admin/services/releasePermissionService';
+
+jest.mock('@admin/services/releasePermissionService');
+
+describe('PublicationAddExistingUsersTab', () => {
+  const testContributors: ManageAccessPageContributor[] = [
+    {
+      userId: 'user-1',
+      userFullName: 'User Name 1',
+      userEmail: 'user1@test.com',
+      releaseRoleId: 'release-role-1',
+    },
+    {
+      userId: 'user-2',
+      userFullName: 'User Name 2',
+      userEmail: 'user2@test.com',
+      releaseRoleId: undefined,
+    },
+    {
+      userId: 'user-3',
+      userFullName: 'User Name 3',
+      userEmail: 'user3@test.com',
+      releaseRoleId: 'release-role-3',
+    },
+    {
+      userId: 'user-4',
+      userFullName: 'User Name 4',
+      userEmail: 'user4@test.com',
+      releaseRoleId: undefined,
+    },
+  ];
+  test('Submits correct userIds to backend', async () => {
+    render(
+      <PublicationAddExistingUsersTab
+        publicationId="publication-id"
+        release={{
+          id: 'release-id',
+          title: 'Test release title',
+          slug: '',
+          approvalStatus: 'Approved',
+          latestRelease: true,
+          live: true,
+          amendment: true,
+          releaseName: '',
+          publicationId: '',
+          publicationTitle: '',
+          publicationSlug: '',
+          timePeriodCoverage: { value: '', label: '' },
+          type: { id: '', title: '' },
+          contact: {
+            id: '',
+            contactName: '',
+            contactTelNo: '',
+            teamEmail: '',
+          },
+          previousVersionId: '',
+          preReleaseAccessList: '',
+        }}
```

Comment:
Rather than mocking this all out, couldn't we use `testRelease` from `pages/release/__data__/testRelease.ts`?